provider/core_dhcp_options_resource.go

```diff
@@ -12,7 +12,7 @@ import (
 func DefaultDHCPOptionsResource() *schema.Resource {
 	return &schema.Resource{
 		Importer: &schema.ResourceImporter{
-			State: schema.ImportStatePassthrough,
+			State: crud.ImportDefaultResource,
```

Comment:
anyway to test these and assert that the "manage_default_resource_id" is set ?

In general, as Brian was saying the other day, our acceptance tests don't actually check for the import scenario passing. So, I am hoping you have manually verified that the state file is updated as you would expect it to.